docs/sources/troubleshoot/debug.md

```diff
@@ -112,6 +112,36 @@ Supported components:
 * `prometheus.relabel`
 {{< /admonition >}}
 * `discovery.relabel`
```

Comment:
We can probably abbreviate this to `discovery.*` unless there's discovery components not added by this PR.